src/Resources/config/routing.yml

```diff
@@ -0,0 +1,11 @@
+lakion_elastic_search_shop_product_index:
+    path: /products
+    methods: [GET]
+    defaults:
+        _controller: lakion_sylius_elastic_search.controller.search:filterAction
+        _sylius:
+            template: "@LakionSyliusElasticSearch/Product/index.html.twig"
+            resource_class: Sylius\Component\Core\Model\Product
+            filter_set: products_option
```

Comment:
Better example of a filter set is something like "t_shirts" or "tvsets" rather than "products_options".